docs/antrea-network-policy.md

```diff
@@ -480,6 +483,10 @@ an `appliedTo` must resolve to. More information on ClusterGroups can be found [
 "sources" or `egress` "destinations". These should be cluster-external IPs,
 since Pod IPs are ephemeral and unpredictable.
 
+**fqdn**: This selector is applicable only to the `to` section in an `egress` block. It is used to
+restrict `egress` access to Fully Qualified Domain Names (FQDN) specified either by exact name or
```

Comment:
restrict is not 100% accurate, as policies can either allow traffic or deny it

I would say something like "It is used to select Fully Qualified Domain Names (FQDNs), specified either by exact name or with a wildcard expression, when defining `egress` rules."